app/src/ui/app.tsx

```diff
@@ -583,6 +585,27 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.setAppMenuState(menu => menu.withReset())
   }
 
+  private onTitlebarDoubleClick() {
+    if (__DARWIN__) {
+      const actionOnDoubleClick: AppleActionOnDoubleClickPref = remote.systemPreferences.getUserDefault('AppleActionOnDoubleClick', 'string')
+      const mainWindow: Electron.BrowserWindow = remote.getCurrentWindow()
+
+      if (mainWindow) {
```

Comment:
I don't think we need to check this, the contract for `getCurrentWindow` states that it always returns a `BrowserWindow` instance when invoked from a renderer. We rely on this elsewhere.